### ISSUE TITLE
New tabs won't open without screen if tmux persistence is enabled

### ISSUE THREAD

**Issue by 5paceToast** [Eugeny/terminus#228]
Steps to reproduce:
1. Install terminus and tmux
2. Ensure screen is not installed
3. In settings -> terminal, change session persistence to "tmux"
  notes:
    - other options should be "off" (notice lack of the "screen" option)
    - while doing this, terminus will attempt to call `which screen`, and fail to find it (as expected)
4. Close the settings page
5. Press "New Terminal" in the startup screen

Expected behavior: a new terminal tab opens
Actual behavior: nothing happens (including no output)

Extra details:
- Platform: Arch Linux
- [Package Used](https://aur.archlinux.org/packages/terminus-terminal/) (AUR, .deb release-based)

**Comment by Eugeny** [comment]
Could you please capture the debug output with the latest version?

**Comment by 5paceToast** [comment]
Attached:
- brand new [log.txt](https://github.com/Eugeny/terminus/files/1800802/log.txt)

Output in the terminal during runtime:
```
Host startup: 71ms
which: no screen in (/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/toast/langs/go/bin:/home/toast/langs/nim/bin:/home/toast/langs/rust/bin:/home/toast/langs/cxx/bin:/home/toast/bin)
info: v 1.0.0-alpha.40
info: No updates
which: no screen in (/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/toast/langs/go/bin:/home/toast/langs/nim/bin:/home/toast/langs/rust/bin:/home/toast/langs/cxx/bin:/home/toast/bin)
info: Quitting
```

config.yml: (apparently, .yml is not a "supported file type")
```
ssh: {}
hotkeys: {}
terminal:
  persistence: tmux
appearance: {}
```

When trying to open a new tab (and failing), in devtools console:
```
ERROR Error: Uncaught (in promise): [object Object]
    at resolvePromise (/opt/Terminus/resources/app.asar/node_modules/zone.js/dist/zone.js:770)
    at /opt/Terminus/resources/app.asar/node_modules/zone.js/dist/zone.js:696
    at rejected (css-base.js:75)
    at ZoneDelegate.invoke (/opt/Terminus/resources/app.asar/node_modules/zone.js/dist/zone.js:391)
    at Object.onInvoke (ng_zone.ts:296)
    at ZoneDelegate.invoke (/opt/Terminus/resources/app.asar/node_modules/zone.js/dist/zone.js:390)
    at Zone.run (/opt/Terminus/resources/app.asar/node_modules/zone.js/dist/zone.js:141)
    at /opt/Terminus/resources/app.asar/node_modules/zone.js/dist/zone.js:818
    at ZoneDelegate.invokeTask (/opt/Terminus/resources/app.asar/node_modules/zone.js/dist/zone.js:424)
    at Object.onInvokeTask (ng_zone.ts:288)
defaultErrorLogger @ errors.ts:42
ErrorHandler.handleError @ error_handler.ts:69
next @ application_ref.ts:367
schedulerFn @ event_emitter.ts:99
SafeSubscriber.__tryOrUnsub @ Subscriber.ts:252
SafeSubscriber.next @ Subscriber.ts:204
Subscriber._next @ Subscriber.ts:135
Subscriber.next @ Subscriber.ts:95
Subject.next @ Subject.ts:61
EventEmitter.emit @ event_emitter.ts:80
onHandleError @ ng_zone.ts:319
ZoneDelegate.handleError @ /opt/Terminus/resources/app.asar/node_modules/zone.js/dist/zone.js:395
Zone.runGuarded @ /opt/Terminus/resources/app.asar/node_modules/zone.js/dist/zone.js:157
_loop_1 @ /opt/Terminus/resources/app.asar/node_modules/zone.js/dist/zone.js:653
api.microtaskDrainDone @ /opt/Terminus/resources/app.asar/node_modules/zone.js/dist/zone.js:662
drainMicroTaskQueue @ /opt/Terminus/resources/app.asar/node_modules/zone.js/dist/zone.js:592
ZoneTask.invoke @ /opt/Terminus/resources/app.asar/node_modules/zone.js/dist/zone.js:490
```

**Comment by Eugeny** [comment]
This has likely now been fixed by the Tmux 1.2 compatibility fix.
